modules/tinymce/src/core/main/ts/keyboard/ContentEndpointNavigation.ts

```diff
@@ -31,12 +31,6 @@ const insertElement = (root: SugarElement, elm: SugarElement, forward: boolean)
   }
 };
 
-const insertBr = (root: SugarElement, forward: boolean) => {
-  const br = SugarElement.fromTag('br');
-  insertElement(root, br, forward);
-  return rangeBefore(br);
-};
-
 const insertBlock = (root: SugarElement, forward: boolean, blockName: string, attrs: Record<string, string>) => {
```

Comment:
Should we rename this to `insertEmptyLine`? As that's more what this is doing and I think that would make this a lot more readable.